src/windows/winjector-helper-core.c

```diff
@@ -380,18 +382,27 @@ initialize_remote_worker_context (RemoteWorkerContext * rwc,
   /* } */
   gum_x86_writer_put_label (&cw, skip_unload_label);
 
+  /* set return 0 */
+  gum_x86_writer_put_xor_reg_reg(&cw, GUM_REG_EAX, GUM_REG_EAX);
+
+  /* if (call GetLastError) { */
+  gum_x86_writer_put_jcc_short_label(&cw, X86_INS_JE, restore_registers, GUM_NO_HINT);
```

Comment:
It should be conditional because we don't know what GetLastError will return in a normal case. When everything went good we don't want to enter this block. The xor is always setting zf to 0 so we won't enter this block is a normal case.

Reply:
But there is no other path to this instruction than the xor, so why isn't it an unconditional jump?

Reply:
You are correct, I'll change it.